src/Components/CreateNewOption/CreateNewOption.tsx

```diff
@@ -12,17 +14,72 @@ const CreateNewOption = () => {
     return (
             <div>
                 <h3>Create New Option</h3>
-                <div className='Toggles'>
-                    <div onClick={()=>setTag("Ans")} id={tag=="Ans" ? "AnsAct" : "Ans"} className="ToggleBtn"><strong>Answer</strong></div>
-                    <div onClick={()=>setTag("Dist")} id={tag=="Dist" ? "DistAct" : "Dist"} className="ToggleBtn"><strong>Distractor</strong></div>
-                </div>
+                <Toggles>
+                    <ToggleBtn onClick={()=>setTag("Ans")} id={tag=="Ans" ? "AnsAct" : "Ans"}><strong>Answer</strong></ToggleBtn>
+                    <ToggleBtn onClick={()=>setTag("Dist")} id={tag=="Dist" ? "DistAct" : "Dist"}><strong>Distractor</strong></ToggleBtn>
+                </Toggles>
                 <div>
-                    <input type="text" placeholder='Suggest an answer or distractor for this question'></input>
+                    <Input type="text" placeholder='Suggest an answer or distractor for this question' />
                     <CategoryInput getCategory={getCategory}/>
-                    <button id="SubmitBtn" disabled>Submit</button>
+                    <SubmitBtn disabled>Submit</SubmitBtn>
                 </div>
             </div>
     )
 }
 
-export default CreateNewOption;
\ No newline at end of file
+const Input = styled.input`
+    padding: 16px;
+    border-radius: 6px;
+    border: 1px solid #bdbdbd;
+    width: 100%;
+    box-sizing: border-box;
+    font-size: 16px;
+    &:placeholder {
+        color: #b7bfc7;
+    }
+    &:focus{
+        outline:none;
+        border-color: #212121;
+    }
+`
+
+const Toggles = styled.div`
+    margin-bottom: 20px;
+`
+
+const SubmitBtn = styled.button`
+    margin-top: 20px;
+`
+const ToggleBtn = styled.div`
```

Comment:
could you add type information to the emotion components that take props?

```suggestion
const ToggleBtn = styled.div<{id: 'Ans' | 'AnsAct' | 'Dist' | ... }>`
```

Reply:
okay!